### PR TITLE
release v4.2.0

### PR DESCRIPTION
**Pull request by Nokel81**
Changes since v4.2.0-rc.3
---

## 🐛 Bug Fixes

* Proxy should listen only on loopback device (#2388) @jakolehm 
* Fix extension command palette loading (#2351) @jakolehm 
* Fix Lens not clearing other KUBECONFIG env vars (#2297) @Nokel81 

## 🧰 Maintenance

* Move clearKubeconfigEnvVars to separate file (#2337) @Nokel81 

Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: static/RELEASE_NOTES.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Here you can find description of changes we've built into each release. While we
 - Fix: Closing workspace menu after clicking on iframe
 - Fix: extension global pages are never able to be visible
 - Fix: recreate proxy kubeconfig if it is deleted
+- Fix: Proxy should listen only on loopback device
+- Fix: Block global path traversal in router
+- Fix: Set initial cursor position for the editor to beginning
+- Fix: Highlight sidebar's active section
 
 ## 4.1.4
```
Comment (jim-docker) [inline static/RELEASE_NOTES.md:47]: shouldn't 4.1.5 be here too?

**Comment by jakolehm** [comment]
PR description is missing.

**Comment by Nokel81** [comment]
@jakolehm What should be in that description nothing has been added since rc.3

**Comment by jakolehm** [comment]
> @jakolehm What should be in that description nothing has been added since rc.3

Maybe just mention that no changes since rc.3.

**Pull request closed without merging**